elastic/kibana
# [Synthetics] omit ssl fields when ssl is disabled

## Summary
Resolves https://github.com/elastic/kibana/issues/149083

Omits SSL fields when TLS is disabled.

### Testing
Testing should be done both in the Uptime and Synthetics app
1. Create a TCP monitor in the Uptime and synthetics app with the following host `8.8.8.8:80`
2. Navigate to the monitor details page for each monitor, either in the Synthetics or Uptime app.
3. Confirm that the url for the monitor is `tcp://8.8.8.8:80`, not `ssl:/8.8.8.8:80`.
